src/main/java/com/fundacionjala/movies/Rental.java

```diff
@@ -0,0 +1,30 @@
+package com.fundacionjala.movies;
+
+class Rental {
+    private final Movie movie;
+    private final int daysRented;
+
+    public Rental(Movie movie, int daysRented) {
+        this.movie = movie;
+        this.daysRented = daysRented;
+    }
+
+
+    public double calculateCharge() {
+    return movie.calculateCharge(getDaysRented());
```

Comment:
Please Give format to all your classes
Take advantage from your IDE
Press ctrl + shift + alt + L